README.md

```diff
@@ -1,2 +1,38 @@
-# Bright Futures Academy
-http://brightfuturesatl.org/the-program/academy/
+# Bright Futures Academy Parent Portal
+This is a web app built to manage payroll at [Bright Futures Academy](
+http://brightfuturesatl.org/the-program/academy/) in Atlanta.
+It provides information for prospective students, and enrolled families
+can sign in to manage their students' tuition and other payments. 
+
+## Getting Started
+### Prerequisites
+You'll need [npm](https://nodejs.org/en/) to run this project.
+
+### Installing
+First, install the packages for both the client and server.
+```
+npm run postinstall
+```
+
+## Running the app
+You can start a development server like this:
+```
+npm run server
+```
+Or you can run the full app like this:
+```
+npm run dev
+```
+
+## Contributing
+All issues are kept under the `Projects` tab. If you find anything you'd like to
+add or see fixed, add it there. 
+
+If you'd like to contribute, create a new branch and make changes as you'd like. 
+Pull requests should to be reviewed by at least one active contributor. 
+Please include screenshots with any frontend changes.
+
+Don't break prod pls.
```

Comment:
Add in:

If you want to test changes that could break prod, we will use a "staging" site that mimics production but won't affect our live site. It is ok if this site is down or breaks as it is used for testing.

1) Log in to Heroku: https://dashboard.heroku.com/
2) Go to bright-futures-academy-staging
3) Go to `deploy`
4) Scroll to the bottom, you should see "Deploy a GitHub branch"
5) Select the branch you'd like to deploy
6) If you want to see logs you can run `heroku login`  and put in the credentials and then `heroku logs --tail --app bright-futures-academy-staging` or see the logs in the console
7) If everything goes well you can see your app on https://bright-futures-academy-staging.herokuapp.com/